akka-docs/src/main/paradox/stream/operators/Sink/collection.md

```diff
@@ -8,11 +8,12 @@
 
 @apidoc[Sink.collection](Sink$) { scala="#collection[T,That](implicitcbf:akka.util.ccompat.Factory[T,Thatwithscala.collection.immutable.Iterable[_]]):akka.stream.scaladsl.Sink[T,scala.concurrent.Future[That]]" }
 
-
 ## Description
 
 @scala[Collect values emitted from the stream into an arbitrary collection `That`. The resulting collection is available through a `Future[That]` or when the stream completes. Note that the collection boundaries are those defined in the `CanBuildFrom` associated with the chosen collection. See [The Architecture of Scala 2.13's Collections](https://docs.scala-lang.org/overviews/core/architecture-of-scala-213-collections.html) for more info. The [`seq`](seq.html) operator is a shorthand for `Sink.collection[T, Vector[T]]`.]@java[Operator only available in the Scala API. The closest operator in the Java API is [`Sink.seq`](seq.html).]
 
+@@@div { .group-scala }
```

Comment:
Hmmm, where does this one end, just after the reactive streams semantics header only to then begin a new scala-group?

All I see removed in the original page is an empty line?

Something weird going on here...

Reply:
It ends on line 25 because AFAIK you can't embed a `div` within a `div`
I agree it seems fishy, but I think it renders what it should

Reply:
Also the second div, has an extra "callout" css class to have the nice outline.
Scala
![image](https://user-images.githubusercontent.com/3337739/101658113-fc4ca280-3a44-11eb-99c4-7e67ef7b0049.png)
Java
![image](https://user-images.githubusercontent.com/3337739/101658168-08386480-3a45-11eb-9e43-aa66098572d4.png)

Reply:
Actually you can by 'nesting' them in more `@@@@` - I'll add a commit with a proposed simplification?

Reply:
TIL

Reply:
Thanks, both of you :)